airbyte-integrations/connectors/source-hibob/source_hibob/manifest.yaml

```diff
@@ -0,0 +1,1189 @@
+version: "0.29.0"
+
+definitions:
+
+  requester:
+    type: HttpRequester
+    url_base: >-
+      {{ 'https://app.sandbox.hibob.com/v1/' if config['is_sandbox'] else
+      'https://app.hibob.com/v1/' }}
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      password: "{{ config[\"password\"] }}"

```

Comment:
This is fine, but would be cleaner if you used single quotes instead of escaping `\"`